build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
Actually, this is kind of telling:

https://github.com/moment/moment/issues/1407

Seems that passing a date-like string into the `moment()` constructor is now discouraged.

```js
console.log([
  moment('April 27, 2015').format(),
  moment('2015-04-27').format()
]);
```

```shell
Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
```